### ISSUE TITLE
Improve error when we detect the host is blocking POST requests on a self hosted site

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#4764]
# Expected behavior

Show a better error message in the full screen dialog, something like:

```
You host is blocking POST requests, the app needs that to communicate to your site.
Contact your host, blablabla.
```

Also, we must add a button the the FAQ page.

# Actual behavior

Full screen dialog with a generic message.

![screenshot-2016-11-09_11 43 30 764](https://cloud.githubusercontent.com/assets/40213/20152137/e3037094-a671-11e6-94a9-e6d556dca852.png)

# Steps to reproduce the behavior

Try to sign in our test instance here: http://do.wpmt.co/405

**Comment by catehstn** [comment]
@mzorz was this closed by #4772 ?

**Comment by mzorz** [comment]
I think yes we should close it now. Thanks @catehstn!
